pkg/cri/io/logger.go

```diff
@@ -194,3 +200,30 @@ func redirectLogs(path string, rc io.ReadCloser, w io.Writer, s StreamType, maxL
 	}
 	logrus.Debugf("Finish redirecting stream %q to log file %q", s, path)
 }
+
+// fastJoin will concatenates buffers with sep like bytes.Join, and then append the result on target.
+func fastJoin(target []byte, bs [][]byte, sep []byte) []byte {
```

Comment:
How about using bytes.Buffer? Would it be slower than fastJoin()?

Reply:
Sure, bytes.Buffer is better. I've changed it.